src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerCommitImage.groovy

```diff
@@ -67,8 +67,10 @@ class DockerCommitImage extends DockerExistingContainer {
 
     @Override
     void runRemoteCommand(dockerClient) {
-        logger.quiet "Commiting image for container '${getContainerId()}'."
-        def commitCmd = dockerClient.commitCmd(getContainerId())
+        def localId = Objects.requireNonNull(getContainerId().getOrNull(), 'containerId must be set')
```

Comment:
IMHO I don't think this handling is necessary. If we expect the container ID property to always container a value then we can just let Gradle fail the build with its own error message.